lib/base.js

```diff
@@ -198,12 +198,12 @@ Base.prototype.spawnCommand = require('./actions/spawn_command');
 /**
  * Prompt user to answer questions. The signature of this method is the same as {@link Inquirer.js}
  *
- * On top of Inquirer.js API, you can provide a `{cache: true}` property for every
- * question descriptors. When true, Yeoman will store/reuser previous user answers as
- * defaults.
+ * On top of the Inquirer.js API, you can provide a `{cache: true}` property for
+ * every question descriptor. When set to true, Yeoman will store/fetch the
+ * user's answers as defaults.
  *
- * @param  {array}   questions  Array of question descriptor objects. See {@link Inquirer.js Documentation}
- * @param  {Function} callback  Receive a question object
+ * @param  {array} [questions]  Array of question descriptor objects. See {@link Inquirer.js Documentation}
```

Comment:
questions is not optional.

Callback could be, but it wouldn't make sense for it to be, so I'd keep it required.